index.html

```diff
@@ -174,14 +174,25 @@ <h3>LET'S CONNECT</h3>
         </section>
 
         <section class="section-3">
-            <h2>Contacts me</h2>
+        <h2>Contacts me</h2>
             <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
-            <div class="input">
-                <input class="input-1" type="text">
-                <input class="input-1" type="text">
-                <input type="text" name="" class="input-2" placeholder="Write your message here">
-                <button class="btn" type="button"><a href="#">Get it Touch</a></button>
-            </div>
+            <form action="#" method="post">
+                <fieldset class="contact-form">
+                    <label for="username">
+                        <input type="text" class="input-contact" required name="username" maxlength="30" placeholder="Username">
+                    </label>
+                    <label for="email">
+                        <input type="email" class="input-contact" required name="email" placeholder="name@domain.com">
+                    </label>
+                    <label for="message">
+                        <textarea name="message" class="msg" maxlength="500" placeholder="Write your message here"></textarea>
```

Comment:
- [ ] Good job applying the `maxlength` attribute to the `textarea` element! 🚀 Please make sure to make this field required too, according to the requirements, all the inputs should be required. 🤓 🔧 

From requirements:

![image](https://user-images.githubusercontent.com/77069011/212985244-3439e1c9-4cb5-479d-9154-bae041f5a311.png)